solidus_braintree.gemspec

```diff
@@ -39,4 +39,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'solidus_dev_support', '~> 2.5'
   spec.add_development_dependency 'vcr'
   spec.add_development_dependency 'webmock'
+
+  spec.post_install_message = "If you're upgrading to v2.0.0, please see the README for upgrade instructions."
```

Comment:
Once you rename `solidus_paypal_braintree` to `solidus_braintree` and run `bundle install`, you would see the post install message:

```
solidus_store 16:56:01 $ bundle install
Fetching https://github.com/solidusio/solidus_braintree.git
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies....
# ...
Using solidus_braintree 1.2.0 from https://github.com/solidusio/solidus_braintree.git (at gsmendoza/96-update-the-extension-namespace-to-solidus-braintree@2ef5bfe)
# ...
Bundle complete! 19 Gemfile dependencies, 131 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
Post-install message from solidus_braintree:
If you're upgrading to v2.0.0, please see the README for upgrade instructions.
```